Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -140,39 +140,34 @@ public async Task PostParticipant(int participantID)
                 //No defined end date for creation
             });
             await _context.SaveChangesAsync();
+            var username = GetAccountByParticipantID(participantID).AD_Username;
+            var participant = GetParticipantByUsername(username);
```

Comment:
perhaps a single service helper method would be helpful here. Something like:
```suggestion
            var participant = GetParticipantByID(participantID);
```

Reply:
Perhaps, but I feel like this helper method isn't really used anywhere else. Moreover, we are still planning to identify with a unique participantID (not gordon_id) and using a secondary key of (ad_username). So I do get what you're saying but eventually we won't even need to use participantID at all.